# ImageDeviation.msg
float32 err_u         # 水平方向像素偏差
float32 err_v         # 垂直方向像素偏差
float32 angle_x        # X轴分解角度
float32 angle_y        # Y轴分解角度